src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -19,13 +19,28 @@ public class CommandResult {
     /** The application should exit. */
     private final boolean exit;
 
+    /** The application should show the task list in the bottom list slot. */
+    private final boolean switchBottomList;
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
     public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
         this.feedbackToUser = requireNonNull(feedbackToUser);
         this.showHelp = showHelp;
         this.exit = exit;
+        this.switchBottomList = false;
+    }
+
+    /**
+     * Constructs a {@code CommandResult} with the specified {@code feedbackToUser},
+     * the specified boolean to show the task list and default values for the other field.
+     */
+    public CommandResult(String feedbackToUser, boolean switchBottomList) {
+        this.feedbackToUser = feedbackToUser;
+        this.showHelp = false;
+        this.exit = false;
+        this.switchBottomList = switchBottomList;
```

Comment:
```suggestion
    public CommandResult(String feedbackToUser, boolean showHelp, boolean switchBottomList, boolean exit) {
        this.feedbackToUser = requireNonNull(feedbackToUser);
        this.showHelp = showHelp;
        this.switchBottomList = switchBottomList;
        this.exit = exit;
    }
```
Perhaps something like that might be more intuitive, and have the called constructors initialize the values, it might get confusing  to have to constructors that are very similar CommandResult(String, boolean, boolean) and a CommandResult(String, boolean).

Reply:
Sounds good, will change this.